frameworks/PHP/clancats/setup.sh

```diff
@@ -0,0 +1,11 @@
+export PHP_HOME=${IROOT}/php-5.5.17
+export COMPOSER_HOME=${IROOT}/php-composer
+export PHP_FPM=${PHP_HOME}/sbin/php-fpm
+export NGINX_HOME=${IROOT}/nginx
+
+sed -i 's|localhost|'"${DBHOST}"'|g' index.php
+sed -i 's|root .*/FrameworkBenchmarks|root '"${TROOT}"'|g' deploy/nginx.conf
```

Comment:
A better fix for this would probably be: `sed -i 's|root /home/ubuntu/FrameworkBenchmarks/;|root '"${TROOT}"'|g' index.php

This way, the value present in your `nginx.conf` is matched directly (no wildcard) and replaced the first time while the second time you run the script, it simply won't find that direct match and sed will replace nothing on this line.